1.10/cli/command-reference/dcos-cluster/index.md

```diff
@@ -24,8 +24,8 @@ dcos cluster
 
 | Command | Description |
 |---------|-------------|
-| [dcos cluster attach](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/)   |   Connect to DC/OS clusters.  |  
-| [dcos cluster list](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-list/)   |  List the clusters that are connect to the DC/OS CLI.  |  
-| [dcos cluster remove](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-remove/)   |  Remove a configured cluster from the DC/OS CLI.   |  
-| [dcos cluster rename](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-rename/)   |  Rename a cluster in the DC/OS CLI.  |  
-| [dcos cluster setup](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-setup/)   |  Configure the CLI to communicate with your DC/OS cluster.  |  
+| [dcos cluster attach](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/)   |  Attach the CLI to a connected DC/OS cluster. |
+| [dcos cluster list](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-list/)       |  List the clusters that are connected to the DC/OS CLI.  |
+| [dcos cluster remove](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-remove/)   |  Remove a cluster from the DC/OS CLI configuration.   |
+| [dcos cluster rename](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-rename/)   |  Rename a cluster in the DC/OS CLI configuration.  |
+| [dcos cluster setup](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-setup/)     |  Connect the DC/OS CLI to a DC/OS cluster.  |
```

Comment:
Done in https://github.com/dcos/dcos-docs/pull/1177/commits/01cf2c23f115d875ccaccad79842c41989bbe957.